types/react-simple-maps/index.d.ts

```diff
@@ -7,6 +7,7 @@
 // TypeScript Version: 2.8
 
 import { GeoPath, GeoProjection } from 'd3-geo';
+import { D3ZoomEvent } from 'd3-zoom';
```

Comment:
```ts
import { D3ZoomEvent } from 'd3-zoom';
```
v1 has dependency to v1 of d3-zoom. You should update this package `tsconfig.json`
to resolve 3 different version of d3-zoom properly:
```json
"paths": {
    "d3-zoom": [
        "d3-zoom/v1"
    ]
},
```

Reply:
done

Reply:
done